api/search.py

```diff
@@ -196,8 +197,12 @@ def as_keywords(t):
     def last_keywords(self):
         return getattr(self, "_last_keywords", [])
 
+    def project(self):
+        return next(iter(self._projects)) if len(self._projects) == 1 else None
```

Comment:
Slight hard to read, maybe:

    return self._projects[0] if len(self._projects) == 0 else None

?